src/pad/src/pad.tcl

```diff
@@ -95,17 +95,35 @@ proc remove_io_bump {args} {
 }
 
 sta::define_cmd_args "assign_io_bump" {-net net \
+                                       [-terminal terminal] \
                                        inst}
 
 proc assign_io_bump {args} {
   sta::parse_key_args "assign_io_bump" args \
-    keys {-net} \
+    keys {-net -terminal} \
     flags {}
   
   sta::check_argc_eq1 "assign_io_bump" $args
 
+  set terminal NULL
+  if { [info exists keys(-terminal)] } {
+    set terms [split $keys(-terminal) {/}]
+    set iterm [lindex $terms end]
+    set inst [join [lrange $terms 0 end-1] {/}]
+
+    set inst [pad::find_instance $inst]
+    set terminal [$inst findITerm $iterm]
```

Comment:
why not dbBlock::findITerm ?

Reply:
I didn't realize that was there. That is a much better solution. Will make that change.